lib/travis/build/script/r.rb

```diff
@@ -0,0 +1,368 @@
+module Travis
+  module Build
+    class Script
+      class R < Script
+        DEFAULTS = {
+          # Basic config options
+          cran: 'http://cran.rstudio.com',
+          warnings_are_errors: false,
```

Comment:
i'm actually torn here -- i'm with @eddelbuettel in principle, but this is like `-Werror` in C -- wouldn't the world be a better place if K&R had just changed that default in the 70s? `;)`